app/views/pages/webhooks.html.md

```diff
@@ -0,0 +1,109 @@
+# Webhook Notifications
+
+## 1. Introduction
+Webhook notifications allow suppliers to subscribe to the PECS API system and receive notifications of when a move record is created or updated. The notification is transmitted via an HTTP POST to an endpoint specified by the supplier. The notification is signed with a shared key to ensure the authenticity of the message. If the endpoint is unavailable then the PECS system will re-attempt the delivery later, retrying up to 25 times.
+
+
+## 2. Subscribing to notifications
+The subscription process is currently manual - please contact the Book a Secure Move via your Slack channel with the following details:
+
+* Supplier name
+* Callback URL: this should be an `https://` endpoint which is publicly accessible and should not require authentication
+* Secret: this is a shared secret which is used to generate a SHA-256 HMAC signature to guarantee the authenticity of the notification
+
+Technical note for the Book a Secure Move team to manually create the subscription record:
+
+* connect to the Staging kubernetes pod and in a rails console, run:
+
+        supplier = Supplier.find_by(key: "<SUPPLIER-KEY>")
+        supplier.subscriptions << Subscription.new(callback_url: "<CALLBACK-URL>", secret: "<SECRET>")
+
```

Comment:
Could this be moved from here into Confluence? This documentation is external-facing, so we don't need to expose internal processes here.